src/main/java/nextstep/ladder/ui/InputView.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.ladder.ui;
+
+import nextstep.ladder.domain.PlayerNames;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final int MIN_MAXLADDERHEIGHT = 1;
+    private static final String INPUT_PLAYERS_COMMENT = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
+    private static final String INPUT_LADDER_HEIGHT_COMMENT = "최대 사다리 높이는 몇 개인가요?";
+    private PlayerNames playerNames;
```

Comment:
상수와 인스턴스 변수 사이에 공백으로 구분하는 것도 가독성을 위해서 좋아보입니다:)
```suggestion

    private PlayerNames playerNames;
```